thirdeye/thirdeye-frontend/app/styles/components/te-navbar.scss

```diff
@@ -0,0 +1,109 @@
+@import "../shared/variable";
```

Comment:
fyi, if this is imported before the component in the app.scss file, you would have access to the variables without having to import them. 
For example:
```
// Utilities
@import "utilities/variables";

// Base Rules
@import "component/tag";
@import "component/navbar";
```

here navbar and tag will have access to the utilities variables